src/main/java/seedu/address/model/Model.java

```diff
@@ -84,4 +84,6 @@
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredTaskList(Predicate<Task> predicate);
+
+    ReadOnlyAddressBook getAddressBook();
```

Comment:
Does this line serve a purpose?

Reply:
It doesn't serve any purpose, IntelliJ helped me add this automatically, but I'll remove it.